From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ken Huang <kenbshuang@google.com>
Date: Fri, 6 Nov 2020 22:21:43 +0800
Subject: ANDROID: drm: expose drm_mode_convert_to_umode to display module

Bug: 139367592
Change-Id: I8473097185aa03df20025b7510633d24a21c758d
Signed-off-by: Ken Huang <kenbshuang@google.com>
(cherry picked from commit 71ca91edd1fe064e40cfbc6129ba5d1f21bbbe2b)
Signed-off-by: Will McVicker <willmcvicker@google.com>
---
 drivers/gpu/drm/drm_modes.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/drm_modes.c b/drivers/gpu/drm/drm_modes.c
--- a/drivers/gpu/drm/drm_modes.c
+++ b/drivers/gpu/drm/drm_modes.c
@@ -1943,6 +1943,7 @@ void drm_mode_convert_to_umode(struct drm_mode_modeinfo *out,
 	strncpy(out->name, in->name, DRM_DISPLAY_MODE_LEN);
 	out->name[DRM_DISPLAY_MODE_LEN-1] = 0;
 }
+EXPORT_SYMBOL_GPL(drm_mode_convert_to_umode);
 
 /**
  * drm_mode_convert_umode - convert a modeinfo into a drm_display_mode
